Dockerfile

```diff
@@ -26,8 +23,7 @@ RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o ma
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
 FROM gcr.io/distroless/static:nonroot
```

Comment:
Should we set specific version of distroless image?

Reply:
AFAIK It doesn't specified mos of the times